### DISCUSSION TITLE
konvers.me down

### DISCUSSION THREAD

**Discussion by catdevnull** [Divested-Mobile/DivestOS-Build#107] [General]
Hi, I can't reach the DivestOS chat room. It seems konvers.me is down.

**Comment by SkewedZeppelin** [comment]
It is not down.
Is your server one of these?
https://github.com/JabberSPAM/blacklist/blob/master/blacklist.txt

**Reply by catdevnull** [reply]
Nope, it's nixnet.services. Pinging konvers.me or conference.konvers.me doesn't work locally either.

**Reply by SkewedZeppelin** [reply]
There are 50 people in the room right now.
And I can ssh into it just fine.

**Reply by SkewedZeppelin** [reply]
It is expected you cannot ping it, it is default drop.

**Reply by catdevnull** [reply]
Okay, contacted Amolith (admin of NixNet.)

**Reply by SkewedZeppelin** [reply]
The strange thing is that I'm connected to a muc hosted on nixnet.

**Reply by SkewedZeppelin** [reply]
In logs I see no errors either, only succcessful connections
```
Accepted outbound s2s EXTERNAL authentication konvers.me -> muc.nixnet.services (hidden_by_ejabberd)
Accepted inbound s2s EXTERNAL authentication muc.nixnet.services -> konvers.me (hidden_by_ejabberd)
```

**Reply by catdevnull** [reply]
Had to close the conversation and try again.

**Comment by catdevnull** [comment]
Had to close the conversation and try again.
